sourcegraph/sourcegraph
# Setup wizard: Add local repositories step

Closes https://github.com/sourcegraph/sourcegraph/issues/47908

This PR adds local repositories UI, our last thing for the Sourcegraph App.

## Test plan
- Run sg start app `EXTSVC_CONFIG_FILE='' sg start app`
- Open `sourcegraph.test:3443/setup` URL
- You should see a local repositories step
- You should be able to use UI to select local repositories 
- After this, the progress bar should be updated (a little bit after, since it takes some time to process just added repositories) 



## App preview:

- [Web](https://sg-web-vova-setup-exp-local-repos-ui.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
